.github/workflows/ci.yml

```diff
@@ -0,0 +1,187 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  release:
+    types: [published]
+  schedule:
+    # Daily at 3:21
+    - cron: '21 3 * * *'
+
+jobs:
+
+  coverage:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macos-latest
+          - windows-latest
+          - ubuntu-latest
```

Comment:
I think we should drop ubuntu from this list.  We have good ubuntu coverage elsewhere.  The CircleCI configuration is probably somewhat more robust than this configuration because it takes extra steps to pin working versions of the dependencies.  Incompatible releases of dependencies frequently broke Tahoe-LAFS CI before that setup was put in place.

The same can be said for macOS and Windows but we *don't* have CI elsewhere that does this kind of dependency pinning so at least the GA setup won't be any worse (and I expect will actually be better, despite not addressing this issue).

Reply:
Dropped Ubuntu from coverage test matrix in 29947aa.  Ubuntu is still in packaging test matrix. I don't think we're running it elsewhere, so maybe this would be useful?